karma.conf.js

```diff
@@ -15,8 +15,8 @@ module.exports = function(config) {
 
     // list of files / patterns to load in the browser
     files: [
-      'https://static.opentok.com/webrtc/v2.2/js/opentok.js',
-      'bower_components/jquery/dist/jquery.js',
+      'https://static.opentok.com/v2/js/opentok.js',
+      'https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js',
```

Comment:
to me, "dependencies" means hard requirements for runtime.

its problematic because when i `bower install` bower has to get a specific version of jQuery and resolve that against any other dependency that wants jquery. this could cause a version conflict. i think the right behavior would be to never have a conflict at `bower install` time, and just let getting it and declaring as a part of the consuming app's dependencies an exercise for the reader.

also, if i use a tool that automatically generates my build based on what was downloaded to my bower_components directory, then i'll end up with jQuery in my build. its common to not want this and to instead grab jQuery from a CDN.